orcestra_book/plans/HALO-20240907a.md

```diff
@@ -62,63 +62,46 @@ from orcestra.flightplan import bco, sal, mindelo, point_on_track, LatLon, IntoC
 
 lon_min, lon_max, lat_min, lat_max = -65, -5, -5, 25
 
-band    = "east"
+band    = "west"
```

Comment:
I still don't get the point of this `band` variable. One could just as well use the actual point (e.g. `sal` or `bco`, or maybe better `tbpb`).

If there's any use of this variable, I'd not only switch the airport, but also the satellite ROI based on it.

Reply:
I inherited it... and don't have any attachment to it